app/services/fees/charge_service.rb

```diff
@@ -378,5 +378,13 @@ def calculate_period_ratio
       ratio = days_passed.fdiv(charges_duration)
       ratio.clamp(0.0, 1.0)
     end
+
+    def filtered_for_charge_boundaries(boundaries)
+      properties = boundaries.to_h
+      properties["fixed_charges_from_datetime"] = nil
+      properties["fixed_charges_to_datetime"] = nil
+      properties["fixed_charges_duration"] = nil
+      properties
+    end
```

Comment:
question... Why do we set fixed charge boundaries in the charge fee as nil (same question applies for the fixed charge fee and the charge boundaries?

Aside of the boundary itself does not make sense at all... we are not doing the same for the subscription fee boundary for charges nor fixed charges fees.